manim/mobject/mobject.py

```diff
@@ -2039,40 +2040,236 @@ def construct(self):
+        col_widths
+            Specific widths for certain columns (from left to right). The given list can contain
+            `None`. In that case the corresponding column will fit its width automatically based
+            on the widest element in that column.
+        flow_order
+            The order in which the grid gets filled with the submobjects. Can be one of the
+            following values: "rd", "dr", "ld", "dl", "ru", "ur", "lu", "ul". For example,
+            "dr" (down right) means: Fill the table column by column downwards, filling the
+            columns rigthwards. "u" and "l" stand for upwards and leftwards.
+
+        .. note:: If either :code:`rows` or :code:`cols` is not defined, it is set to the length of either
+            :code:`row_alignments` / :code:`col_alignments` or :code:`row_heights` / :code:`col_widths` if
+            one of those is defined. If only one of :code:`rows` and :code:`cols` can be determined
+            like this, the other will be choosen big enough to fit all submobjects.
+            If neither :code:`cols` nor :code:`rows` is set implicitly, they will be choosen to be
+            about the same, tending towards :code:`cols > rows` (simply because videos are wider
+            then they are high).
+
+        .. note:: If both :code:`cell_alignment` and either :code:`row_alignments` or :code:`col_alignments` is
+            defined the latter has higher priority in its direction.
+
+
+        Returns
+        -------
+        Mobject
+            The mobject.
+
+        Raises
+        ------
+        ValueError
+            If rows and cols are choosen to small to fit all submobjetcs.
+        ValueError
+            :code:`cols`, :code:`col_alignments` and :code:`col_widths` or :code:`rows`,
+            :code:`row_alignments` and :code:`row_heights` have mismatching sizes.
 
         Examples
         --------
 
-        .. manim:: ArrangeExample
+        .. manim:: ArrangeInGrid
             :save_last_frame:
 
-            class ArrangeExample(Scene):
+            class ArrangeInGrid(Scene):
                 def construct(self):
-                    self.add(*[Square(color= random_bright_color()) for i in range(0,5)])
-                    x = VGroup(*self.mobjects).arrange_in_grid(buff=0.2)
-                    self.add(x)
+                    #Add some numbered boxes:
+                    np.random.seed(3)
+                    boxes = VGroup(*[Rectangle(WHITE, np.random.random()+.5, np.random.random()+.5).add(Text(str(i+1)).scale(0.5)) for i in range(22)])
+                    self.add(boxes)
+
+                    boxes.arrange_in_grid(
+                        buff=(0.25,0.5),
+                        col_alignments="lccccr",
+                        row_alignments="uccd",
+                        col_widths=[2, *[None]*4, 2],
+                        flow_order="dr"
+                    )
+
+
         """
+        from manim import Line
```

Comment:
AFAIK, the default policy for intra-library imports is to use relative imports to the required module, like the suggestion.
```suggestion
        from .geometry import Line
```